grails-app/domain/com/careerfly/organization/interview/Interview.groovy

```diff
@@ -0,0 +1,29 @@
+package com.careerfly.organization.interview
+
+import com.careerfly.organization.Company
+import com.careerfly.taggable.Tag
+import com.careerfly.user.User
+
+class Interview {
```

Comment:
@akshitagangrade @ankit18singh @apoorvsrivastava1116 @Smita007 @atul12594 @shubhamj15 @Sagars09 @PraveenTahalyani

Fix this problem at all classes and pay attention https://gist.github.com/sagrawal14/713a86d52ff862babe42e6e1a5e4140c#file-groovyjavapractices-groovy-L21